lib/fog/aws/requests/data_pipeline/create_pipeline.rb

```diff
@@ -24,14 +24,35 @@ def create_pipeline(unique_id, name, description=nil, tags=nil)
             :body => Fog::JSON.encode(params),
             :headers => { 'X-Amz-Target' => 'DataPipeline.CreatePipeline' },
           })
-
-          Fog::JSON.decode(response.body)
```

Comment:
Ah. I'm open to either way. Not major-bumping is obviously easier. I think this stuff is relatively new, so that makes it feel a bit more flexible to change. But I agree it could break things. What is your inclination?